tasks/cssmin.js

```diff
@@ -41,6 +41,9 @@ module.exports = function (grunt) {
 
       try {
         compiled = new CleanCSS(options).minify(availableFiles);
+        if (compiled.errors && compiled.errors.length) {
+            throw compiled.errors;
```

Comment:
I assume this is an array since it's named `errors`. You should only throw actual errors. Join it with a newline and `grunt.warn` it instead.